cli/contractAddress.txt

```diff
@@ -0,0 +1 @@
+{"VkRegistry":"0xE841a45ecbFfBcA75EBaAf68af29a777ea29Cc74","InitialVoiceCreditProxy":"0xFe9FBDc17d9247ee10c404B3ABD24312938A4D77","SignUpGatekeeper":"0x72fF92B2D4cf20e42563c44C3b53a9d6C0a14a23","Verifier":"0xcBd93e157DC4B2bE0Fe44689284602B33dC7cD4F","MACI":"0xc7B7f06BE87cB7159ffDb09B3A0cceF1Dcf7c8a4","StateAq":"0xA2dDe729c1d3AC0B111eD42E5f166A5777a2a742","PollFactory":"0x5B875838F8c0AE9213df6f994Cf1b6cBcA5f3c7D","TopupCredit":"0xA0A520C14714182D99f9b21e368deD1b70b7CF3a","PoseidonT3":"0x74E0ad7323b81cf92f19E55Da78A3e86aff179E6","PoseidonT4":"0x75D0ef1c00DCe48BD77F0d8E851f8B045dC2099d","PoseidonT5":"0xcc347EA92C67F7c0D1671e2ee3c518F5108cBBE4","PoseidonT6":"0x7B22fE380A2a71f1129C7F4A2984e05E45EC76cA","Verifier-0":"0x290532c242535B05728d3017668866A58a12f223","MessageProcessor-0":"0x72a27863eB52E8b840f52F9F306921f4eF138f83","Tally-0":"0xEFB0c18f85D2602C88d2d28Eb24687c31db8c028","Subsidy-0":"0x91926D8f532b861Fb6326a6305ADFD84E3c3d24F","Poll-0":"0xd47D2A33dD955e36fB80A7Db7dcc157ebE4a3a7f"}
```

Comment:
What's the purpose of these?

Reply:
These were used in the old cli, I added a directive to .gitignore to not commit here: https://github.com/privacy-scaling-explorations/maci/pull/876/files#diff-1d6acef67c4c117ce3cf27106114ac0ab7ac91ffa6fc08159f1798688e02f891 - they should be removed from the repo

Reply:
Ok, will fix